packages/edit-post/src/components/preferences-modal/style.scss

```diff
@@ -1,4 +1,56 @@
 .edit-post-preferences-modal {
+	min-width: 350px;
+	@include break-small() {
+		height: calc(100% - #{$header-height} - #{$header-height});
+	}
+
+	.edit-post-preferences-modal__content {
+		.components-navigation {
+			background-color: #fff;
+			padding: 0;
+			.components-navigation__menu {
+				margin: 0;
+				.components-navigation__item {
+					& > button {
+						color: initial;
+						padding-left: 0;
+						padding-right: 0;
+					}
+					.components-toggle-control__label {
+						color: initial;
+					}
+				}
+				.components-navigation__menu-title-heading {
+					color: initial;
+					border-bottom: 1px solid $gray-300;
+					padding-left: 0;
+					padding-right: 0;
+				}
+			}
+		}
+	}
+
+	.edit-post-preferences__tabs {
+		display: flex;
+		flex-direction: row;
+		.components-tab-panel__tabs {
+			width: 200px;
+			.components-tab-panel__tabs-item {
+				&.is-active {
+					background: $gray-100;
+					box-shadow: none;
+				}
+				&:focus:not(:disabled) {
+					box-shadow: none;
+				}
+			}
+		}
+		.components-tab-panel__tab-content {
+			width: 400px;
+			padding-left: $grid-unit-30;
+		}
+	}
```

Comment:
It feels like there's too much CSS needed for this. Were the base components styles (TabPanel and Navigation) not enough to achieve the desired result? Do we want to move some of these styles to the generic component or are  these specific to the   Preferences modal? Do we need the styles to be different from the original ones that come from the UI components?

Reply:
Unfortunately there will be more css changes :) - I wanted to get the first feedback. `Navigation` component was built for FSE with a dark theme so it's not fitting really well here. But since we're proceeding with this approach I'll explore the best place to put/change styles.

Reply:
> Navigation component was built for FSE with a dark theme so it's not fitting really well here.

Yeah, for instance this makes me wonder if the Navigation component need to have dark styles :). Ideally, it just works no matter its container background, alternatively it could support both with a prop. `isLight/isDark`

Reply:
That's what I'm thinking as well!

Reply:
I think @ItsJonQ already had a light / dark version of this, which is going to be used (as light) in the global styles case.

Reply:
For context, light/dark mode is built into the Navigator component from the [new component system](https://github.com/ItsJonQ/g2/tree/master/packages/components/src/Navigator).

The one we currently have in `@wordpress/components` isn't using that yet.

Reply:
Regarding the css in `Tabs`, I think it's okay to leave it here, based on Joen's comments:
>We don't actually use this gray active style anywhere else
>I think we can probably go with the gray background for now, and we can explore if/when we get a better pattern.

Regarding the `Navigation` css, I'm not sure how to properly integrate it in the component with `@emotion` (css-in-js).
Help with that and a sanity check on css rules would be really appreciated! @jasmussen, @ItsJonQ